extensions/amp-list/0.1/amp-list.js

```diff
@@ -251,6 +362,21 @@ export class AmpList extends AMP.BaseElement {
       });
     }
 
+    if (this.loadMoreEnabled_) {
+      this.mutateElement(() => {
+        this.initializeLoadMoreButton_();
+        this.initializeLoadMoreLoadingElement_();
+        this.initializeLoadMoreFailedElement_();
+        this.initializeLoadMoreEndElement_();
+        // Hide overflow element
+        const overflowElement = this.getOverflowElement();
+        if (overflowElement) {
+          toggle(overflowElement, false);
+        }
+      }).then(() => {
+        this.adjustContainerForLoadMoreButton_();
```

Comment:
This running at build time instead of layout time was the culprit of the media query issue.